.github/workflows/fmt-check.yml

```diff
@@ -0,0 +1,40 @@
+name: Quick check Formatting
```

Comment:
I would suggest aligning the `name` and the filename.
This workflow used to be called `quick-check.yml`.

Reply:
But it's not clear from the name "quick check" what the action does. Maybe `quick-fmt-check` then?